src/main/gov/nasa/jpf/jvm/JVMClassInfo.java

```diff
@@ -150,7 +150,8 @@ public void setBootstrapMethod (ClassFile cf, Object tag, int idx, int refKind,
         // The check order cannot be reversed since other BSM may not
         // have forth argument in `cpArgs`
         boolean isSerializable = false;
-        if (mth.equals("altMetafactory")) {
+        if (cls.replace('/', '.').equals("java.lang.invoke.LambdaMetafactory")
```

Comment:
if (cls.equals("java/lang/...") ...
// (I think your version is perhaps more readable, but for people who know bytecode, / as package name separators is nothing new, so it's better to avoid a search/replace when it's not needed